code/WorkInProgress/multiContext.dm

```diff
@@ -278,10 +278,11 @@ var/list/globalContextActions = null
 				if(action) applicable.Add(C)
 
 		var/obj/item/W = src.equipped()
-		if(W && W != target && length(W.contextActions))
-			for(var/datum/contextAction/C in W.contextActions)
-				var/action = C.checkRequirements(target, src)
-				if(action) applicable.Add(C)
+		if(W && !(W.object_flags & IGNORE_CONTEXT_CLICK_EQUIPPED))
+			if(W != target && length(W.contextActions))
+				for(var/datum/contextAction/C in W.contextActions)
```

Comment:
as() is applicable I think

Reply:
hmmmm...i'm not entirely certain how much of the rest of multicontext code that would touch...i didnt code that line myself and multicontext menus seem a little spaghetti at the moment...

Reply:
(it's definitely applicable, i just refreshed/cleaned up/sped up the code of multicontext personally)

Reply:
the latest version of the moved file now uses as() so this point is now resolved :D